cookbooks/shell_3d_postprocess.prm

```diff
@@ -0,0 +1,106 @@
+# A description of convection in a 3d spherical shell with 
+# a prescribed initial condition based on a harmonic
+# perturbation
+
+# Define the number of space dimensions we would like to 
+# work in:
+set Dimension                              = 3 
+
+# Specify the time you want to let the model run for in 
+# years and the output directory. Here we only calculate
+# the instantaneous solution.
+set End time                               = 0
+set Use years in output instead of seconds = true
+set Output directory                       = output_s2
```

Comment:
usually cookbooks use an output directory of the form: output-prmname, i.e. output-shell_3d_postprocess in this case. This is mostly to avoid cookbooks overwriting each other, which should not happen with your name. Still for conformity please update.